supervisor.py

```diff
@@ -138,7 +90,7 @@ def wait(self):
 
     # Try to pull
     print("We're going to update your instance of redbot now.")
-    updates = subprocess.Popen(['git', 'pull'])
+    updates = subprocess.run(['git', 'pull'])
     if updates.wait() != 0:
         print("Error updating Redbot. Please investigate.")
         sys.exit(0)
```

Comment:
Yeah that's shorted, but exit status should be 0. Git had an error, but the manager itself didn't exit with an error.